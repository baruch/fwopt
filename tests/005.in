-A INPUT --source-port 1900 -j DROP
