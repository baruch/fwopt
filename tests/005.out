Setting source port but protocol is not udp nor tcp
Failed parsing input
